client/types.go

```diff
@@ -365,6 +365,51 @@ type AccountList struct {
 	Accounts []AccountElem `json:"accounts"`
 }
 
+type BillingAddressSpec struct {
+	Company     *string `json:"company,omitempty"`
+	Address     *string `json:"address,omitempty"`
+	Address1    *string `json:"address1,omitempty"`
+	Address2    *string `json:"address2,omitempty"`
+	PhoneNumber *string `json:"phone_number,omitempty"`
+	City        *string `json:"city,omitempty"`
+	Country     *string `json:"country,omitempty"`
+	State       *string `json:"state,omitempty"`
+	Zip         *string `json:"zip,omitempty"`
+}
+
+type DeveloperAccountItem struct {
```

Comment:
On account creation org_name, username, email, password and id are required.
I guess then this API does not check that right? we just let System API return an error in that case, we don't enforce parameters at the porta-go-client API level

Reply:
In the account creation (aka `signup`) the use `Params` to sent account info. The response returned will be parsed to this `DeveloperAccountItem`. We can expect OrgName to be there in the response. However, this struct is used to do update operations as well, and OrgName is optional param, so it could be empty.